mis_builder_analytic_filter/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "MIS Builder Analytic Axis Filter",
+    'version': '8.0.1.0.0',
+    'category': 'Reporting',
+    'summary': """
```

Comment:
The description in the readme is not more informative either. It's still nicer to have a short summary that rephrases the module name. I'm open to suggestions.